<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="UsernameEntry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>360</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterBut">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>700</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBackBut">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>710</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmPasswordEntry">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmPasswordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEntry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>400</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowPasswordBut">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>350</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
